pkg/watcher.go

```diff
@@ -191,6 +192,15 @@ func (w *Watcher) Start() error {
 				w.Error().Msg("error casting file string from register")
 				return false
 			}
+
+			// Exclude file from monitoring if it belongs to exclusion list
+			for _, excludeFile := range w.Excludes {
```

Comment:
It is the initial exclusion, also we need dynamic exclusion while receiving creation events (directory/file/renaming). It can be added to `addInode` function. It can be an additional commit

Reply:
Moved this function for dynamic watching